FSD-6 Engineers Performance Calculator

AMubinshaikh opened the pull request:
FSD 6 is based on Array logic .This  Program calculate engineers performance based on their  speed and efficiency.

AMubinshaikh committed:
FSD-6 is completed Arrays based

FSD 6 is based on Array logic .This  Program calculate engineers performance based on their  speed and efficiency.

abdulmuheet commented on FSD6-arrays-Engineer/Program.cs:
```diff
@@ -0,0 +1,16 @@
+    {
+        static void Main(string[] args)
+        {
+            EngineersEfficiency engineersEfficiency= new EngineersEfficiency();
```
Instead of using the Console.ReadLine for inputs values, pass it from here with a different set of examples mentioned in the jira case.

abdulmuheet commented on FSD6-arrays-Engineer/Program.cs:
```diff
@@ -0,0 +1,16 @@
+            engineersEfficiency.AcceptEngineerDetails();
+            engineersEfficiency.CalculatePerformance();
+            engineersEfficiency.CalculateResult();
+            engineersEfficiency.DisplayResult();
```
Display logic should be written here after fetching it from function.

abdulmuheet reviewed (changes requested):
Do check the comments provided my the PR#8 as most of them are also applicable in this PR as well.

AMubinshaikh committed:
Changes are done

FSD-6 changes are done
changes:
pass data from function and display multiple type reult as given in JIRA board.
and display data in Main function. kindly re check

AMubinshaikh committed:
changes are done. its not taking input from user

Commits for fsd6 which is calculate performance of engineers efficiency
removing taking input from user and generate result based on parameters

AMubinshaikh committed:
Indentation and removing extra lines are done

Indentation and removing extra lines are done.

abdulmuheet commented on FSD6-arrays-Engineer/Program.cs:
```diff
@@ -0,0 +1,29 @@
+            int numberOfEngineers = 2;
+
+            // # 2:
+            int[] speedList = { 2,10,3,1,5,8 };
```
Keep it commented to have default code in working condition.

abdulmuheet commented on FSD6-arrays-Engineer/Program.cs:
```diff
@@ -0,0 +1,29 @@
+            int numberOfEngineer = 4;
+
+            EngineersEfficiency engineersEfficiency = new EngineersEfficiency ();
+
```
remove extra line break

abdulmuheet reviewed (changes requested):
.dll is not required as communicated.

AMubinshaikh committed:
Update Program.cs

AMubinshaikh committed:
Delete FSD6.dll
